test/next/test/@vercel/blob/index.test.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+import { type BlobResult } from '@vercel/blob';
+
+test.describe('@vercel/blob', () => {
+  test.describe('api', () => {
+    [
+      'vercel/blob/api/app/body/edge',
+      'vercel/blob/api/app/body/serverless',
+      'api/vercel/blob/edge',
+      'api/vercel/blob/serverless',
+    ].forEach((path) => {
+      test(path, async ({ request }) => {
+        const data = (await request
+          .post(`${path}?filename=test.txt`, {
+            data: 'Hello world',
+          })
+          .then((r) => r.json())) as BlobResult;
+        expect(data.contentDisposition).toBe('attachment; filename="test.txt"');
+        expect(data.contentType).toBe('text/plain');
+        expect(data.pathname).toBe('test.txt');
```

Comment:
Should we add a quick test that shows, when downloaded, the blob content is the one we expect?